packages/api/definitions/moderation-behaviors.d.ts

```diff
@@ -0,0 +1,48 @@
+import type { LabelPreference } from '../src'
+
+export interface ModerationBehaviorResult {
+  cause?: string
+  filter?: boolean
+  blur?: boolean
+  alert?: boolean
+  noOverride?: boolean
+}
+
+export interface ModerationBehaviorScenario {
+  cfg: string
+  subject: 'post' | 'profile' | 'userlist' | 'feedgen'
```

Comment:
Would it be equally valid to use collection names here, and this is just a more convenient interface?  Or would using collections end-up being constraining/incorrect in some way?

Reply:
yeah, this seems like it would be "more correct" and future-proof, if a bit more verbose.

Reply:
This `d.ts` is just used informally at the moment and is only used by internal code. We can expand it in the future if it becomes more official